bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
     olm.skipRange: '>=4.0.0 <5.0.0'
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
-    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.14.0+git
```

Comment:
Is this change needed for `make lint` to succeed?

Reply:
It seems like it is.
If I revert the change by adding another commit:
```
[sebsoto@sdesktop windows-machine-config-operator]$ git show HEAD
commit c2a66d4383a37d4075c4e1f96b54ca10f18fc299 (HEAD -> lint_bundle)
Author: Sebastian Soto <ssoto@redhat.com>
Date:   Fri Dec 17 12:36:33 2021 -0500

    f

diff --git a/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml b/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
index de7c8b62..cab8700e 100644
--- a/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
+++ b/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
@@ -11,7 +11,7 @@ metadata:
     olm.skipRange: '>=4.0.0 <5.0.0'
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
-    operators.operatorframework.io/builder: operator-sdk-v1.14.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/openshift/windows-machine-config-operator
     support: Red Hat
[sebsoto@sdesktop windows-machine-config-operator]$ hack/verify-bundle.sh 
/home/sebsoto/code/openshift/windows-machine-config-operator/bin/controller-gen rbac:roleName=windows-machine-config-operator crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
/usr/local/bin/operator-sdk generate kustomize manifests -q
WARN[0000] Skipping definitions parsing: APIs root dir "api" does not exist 
cd config/manager && /home/sebsoto/code/openshift/windows-machine-config-operator/bin/kustomize edit set image controller=REPLACE_IMAGE
/home/sebsoto/code/openshift/windows-machine-config-operator/bin/kustomize build config/manifests | /usr/local/bin/operator-sdk generate bundle --overwrite=false -q --version 5.0.0 --channels="preview,stable" --default-channel="stable"
sed -i 's/REPLACE_IMAGE:latest/REPLACE_IMAGE/g' bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
/usr/local/bin/operator-sdk bundle validate ./bundle
INFO[0000] All validation tests have completed successfully 
sed -i 's/windows-machine-config-operator\.v.\.0\.0/windows-machine-config-operator.v5.0.0/g' ./bundle/windows-machine-config-operator.package.yaml
bundle lint failure
 M bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
```

Reply:
My question was to do with the whole commit. Moreover `operator-sdk-v1.14.0+git` seems wrong given the SDK version we are using.